db/migrate/20160302203922_remove_is_datacenter_from_ems_folder.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveIsDatacenterFromEmsFolder < ActiveRecord::Migration[5.0]
+  class RemoveIsDatacenterFromEmsFolder::EmsFolder < ActiveRecord::Base
```

Comment:
You're in the RemoveIsDatacenterFromEmsFolder namespace so you can just call this EmsFolder.